tests/test_rtcicetransport.py

```diff
@@ -190,19 +198,34 @@ def test_invalid_uri(self):
     def test_stun(self):
         uri = parse_stun_turn_uri("stun:stun.services.mozilla.com")
         self.assertEqual(
-            uri, {"host": "stun.services.mozilla.com", "port": 3478, "scheme": "stun"}
+            uri,
```

Comment:
gratuitous change now that we"re not testing "protocol", ditto for the following tests